lang/ilias_pl.lang

```diff
@@ -2727,7 +2726,7 @@ chatroom_adm#:#client_settings#:#Ustawienia ogólne
 chatroom_adm#:#enable_osd#:#Czat na forum: Aktywuj powiadomienia on-screen
 chatroom_adm#:#general_settings_title#:#Ogólne ustawienia dotyczące czatu
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Interwał zapytań (sekundy)
+chatroom_adm#:#osd_interval#:#Interwał zapytań (sekundy)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.